src/phone-number-input/phone-number-input-helper.tsx

```diff
@@ -81,7 +81,7 @@ export namespace PhoneNumberInputHelper {
             },
             {
                 formattedText: "",
-                remainingText: text.split(""),
+                remainingText: text ? text.split("") : "",
```

Comment:
the logic still works with string, but for correctness, it should be initialised to an empty array
```suggestion
                remainingText: text ? text.split("") : [],
```